test/daemon/daemon.go

```diff
@@ -23,6 +23,7 @@ const (
 	Listen        = "unix:///tmp/test/pouch/pouchd.sock"
 	ContainerdAdd = "/tmp/test/pouch/containerd.sock"
 	ListenCRI     = "unix:///tmp/test/pouch/pouchcri.sock"
+	Pidfile       = "/tmp/pouch.pid"
```

Comment:
How about change default Pidfile to `/tmp/pouch/pouch.pid`, so we may easily remove legacy files by removing directory `/tmp/test/pouch`